src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1874,11 +1874,9 @@ private float GetCompoundAmount(Patch patch, string compoundName)
             case "nitrogen":
                 return patch.Biome.Compounds[compound].Dissolved * 100;
             case "iron":
-                return patch.GetTotalChunkCompoundAmount(compound);
+                return patch.GetTotalChunkCompoundAmount(compound) * patch.Biome.SpawnRateMultiplier;
             default:
-                return patch.Biome.Compounds[compound].Density *
-                    patch.Biome.Compounds[compound].Amount + patch.GetTotalChunkCompoundAmount(
-                        compound);
+                return patch.Biome.Compounds[compound].Density * patch.Biome.SpawnRateMultiplier;
```

Comment:
You changed the logic here, why?

Reply:
When playtesting, it felt like the percentages were "off", like if compounds have the same density you have equal chances of running into each cloud. But if they have different Amounts, then the percentages vary.

Reply:
But this is the *chunk* amount. Which heavily depends on how much resources each chunk has as the player can engulf it to get all of it's resources.